pubspec.lock

```diff
@@ -0,0 +1,317 @@
+# Generated by pub
```

Comment:
This file is committed intentionally as suggested by the [Melos docs](https://melos.invertase.dev/getting-started#install-melos-in-the-workspace).

> The corresponding pubspec.lock file should also be committed. Make sure to exclude it from the .gitignore file.